app/src/components/points-of-interest/IAPP/IAPP-Site.tsx

```diff
@@ -42,17 +42,19 @@ export interface IAPPSitePropType {
 export const IAPPSite: React.FC<IAPPSitePropType> = (props) => {
   const classes = useStyles();
 
-  const {site_id, map_sheet, slope, aspect, specific_use, soil_texture, surveys, mechanical_treatments, comments}
+  const {site_id, map_sheet, aspect, specific_use, soil_texture, surveys, mechanical_treatments, comments}
      = props?.record?.point_of_interest_payload?.form_data?.point_of_interest_type_data;
   const {access_description, created_date_on_device}
      = props?.record?.point_of_interest_payload?.form_data?.point_of_interest_data;
   const paper_file = props?.record?.point_of_interest_payload?.form_data?.point_of_interest_data?.paper_file[0]?.description;
   const longitude = props?.record?.point_of_interest_payload?.geometry[0]?.geometry?.coordinates[0];
   const latitude = props?.record?.point_of_interest_payload?.geometry[0]?.geometry?.coordinates[1];
+  const {slope, elevation} = props?.record?.point_of_interest_payload?.geometry[0].properties;
   const {Jur1, Jur1pct, Jur2, Jur2pct, Jur3, Jur3pct} = surveys ? surveys[0] : {Jur1: 'Not Specified', Jur1pct: '100', Jur2: '', Jur2pct: '0', Jur3: '', Jur3pct: '0'};
   // Tester: {Jur1:'A', Jur1pct:'50', Jur2:'B', Jur2pct:'20', Jur3:'C', Jur3pct:'30'};
 
   const ifApplicable = (value) => (value && String(value).trim() != '') ? value : <div className={classes.missingValue}>N/A</div>;
+  const toDecimalPlaces = (n, decimals = 4) => Number.parseFloat(n).toFixed(decimals);
```

Comment:
question: can this just be:

```suggestion
  const toDecimalPlaces = (n, decimals = 4) => parseFloat(n).toFixed(decimals);
```

also, I have been using 2 decimal places for lat/lng and area so far so we should confirm what they want and stick to it :)

Reply:
hah!  nice, never realized that was global

Yeah it would appear a 4-decimal area seems to be the desired standard (with hectares at least).  Haven't gotten any complaints on lat/lon, but I'd kinda prefer those to have a set number of decimals too rather than the raw dumps I'm using here... 2 decimals, you say?  Works for me